app/SmartFlowUI/backend/MinimalApi.csproj

```diff
@@ -65,14 +65,21 @@
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Azure.Search.Documents" />
     <PackageReference Include="Azure.Storage.Blobs" />
+    <PackageReference Include="Blazor.LocalStorage.WebAssembly" />
+    <PackageReference Include="Blazor.SessionStorage.WebAssembly" />
     <PackageReference Include="Markdig" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" />
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" />
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" />
     <PackageReference Include="Microsoft.Azure.Cosmos" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+    <PackageReference Include="Microsoft.Extensions.Http" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime" />
     <PackageReference Include="Microsoft.SemanticKernel" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" />
+    <PackageReference Include="MudBlazor" />
     <PackageReference Include="Newtonsoft.Json" />
```

Comment:
might be a change for another PR but we should remove Newtonsoft

Reply:
I tried that - there was one place that used Newtonsoft (not my new code), and when I switched it over, it broke, so I left it.